test/system/505-networking-pasta.bats

```diff
@@ -10,6 +10,9 @@
 load helpers
 load helpers.network
 
+# All tests in this file must be able to run in parallel
+# bats file_tags=ci:parallel
```

Comment:
Judgment call. Adding a `test_tags=` comment to every single `@test` would IMO be uglier. Cost is having to move one test outside this module.